css/style.css

```diff
@@ -0,0 +1,44 @@
+#main {
+    background-color: #EEEEEE;
+}
+#title {
+    margin: auto;
+    max-width: 50%;
+    text-align: center;
+    
+}
```

Comment:
In line number 8, the blank line is unnecessary. Please avoid using blank lines so that the entire code base is consistent and readable.

Reply:
Great review💯. I think you can also include emojis ☺ and gifs to make the code review more human-friendly😎